### PR TITLE
Update proposal link to point to archive

### PR DESCRIPTION
**Pull request by voelzmo**
Original link says
```
Design proposals have been archived.

To view the last version of this document, see the Design Proposals Archive Repo.

Please remove after 2022-04-01 or the release of Kubernetes 1.24, whichever comes first.
```

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ flag.
 It is recommended that the kubernetes system daemons are placed under a top
 level control group (`runtime.slice` on systemd machines for example). Each
 system daemon should ideally run within its own child control group. Refer to
-[the design proposal](https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md#recommended-cgroups-setup)
+[the design proposal](https://git.k8s.io/design-proposals-archive/node/node-allocatable.md#recommended-cgroups-setup)
 for more details on recommended control group hierarchy.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/reserve-compute-resources.md:95]: @voelzmo, a better fix would be to reproduce that recommended hierarchy as a diagram (perhaps [using Mermaid](https://www.k8s.dev/blog/2021/12/01/improve-your-documentation-with-mermaid.js-diagrams/)). Would you be willing to do that?

If that looks like more work than you'd be happy to take on, we can log a new issue. The new issue would be about adding that diagram into the docs and updating https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/ to link to the page with that diagram.

**Comment by kbhawkey** [comment]
Page preview:

https://deploy-preview-31816--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/reserve-compute-resources/#kube-reserved

**Comment by kbhawkey** [comment]
Thanks @voelzmo .
/lgtm

**Pull request merged**